client/src/components/editor/RichTextEditor.css

```diff
@@ -1,21 +1,55 @@
+:root {
+  --banner-height: 50px
+}
 .editor-container {
   border: 1px solid #ccc;
   border-radius: 4px;
 }
 
+/* Style used to make rich-text-editor to full-size */
+.editor-container-fullsize {
+  position: absolute; 
+  /* In full size mode the richtext-editor start position, just below the topbar */
+  top: var(--banner-height);
+  left: 0;
+  max-width: none;
+  max-height: calc(100% - var(--banner-height));
+  width: 100%;
+  height: 100%;
+  background: whitesmoke;
+  z-index: 1201;
+  /* Hide scrollbars in both x and y directions */
+  overflow-y: hidden;
+  overflow-x: hidden;
+}
+
 .toolbar {
   width: 100%;
   display: flex;
   flex-wrap: wrap;
+  /* position, top and z-index with the following values are used to make rich text toolbar alway accesible even if the text is getting longer */
+  position: sticky;
+  top: 0;
+  z-index: 1;
+  border: 1px solid #ccc;
 }
 
 .editor-container .editable {
   min-height: 10rem;
-  max-height: 50rem;
+  max-height: 25rem;
   overflow: auto;
   padding: 1rem;
 }
 
+/* Style used to make editable area to full-size */
+.editor-container .editable-fullsize {
+  /* Prevent edit-area from being longer than the editor-container */
+  max-height: calc(100% - var(--banner-height)); 
```

Comment:
It looks like this can simply be:
```
  max-height: 100%; 
```

Reply:
must be limited otherwise it doesn't work